unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Billboard/BillboardsController.cs

```diff
@@ -0,0 +1,88 @@
+using DCL;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BillboardsController : IBillboardsController
+{
+    private readonly List<Billboard> billboards = new List<Billboard>();
+    private Coroutine updateCoroutine;
+
+    private Vector3Variable CameraPosition => CommonScriptableObjects.cameraPosition;
+    private Vector3 lastCamPosition;
+    private bool camUpdated;
+
+
+    public static BillboardsController Create()
+    {
+        BillboardsController controller = new BillboardsController();
+
+        return controller;
+    }
+
+    public void Initialize()
+    {
+        updateCoroutine = CoroutineStarter.Start(UpdateCoroutine());
+    }
+
+    public void BillboardAdded(IBillboard inBillboard)
+    {
+        Billboard billboard = inBillboard as Billboard;
```

Comment:
Why cast it back to Billboard? This breaks the Liskov substitution principle, the idea of having an interface is to not deal with the implementation itself. This means that we could have different billboards in the future that use the same interface and we won't have to deal with this implementation of BillboardsController.

For that to work, IBillboard should have the definitions for the functions that we need, in this case:
- You could replace [these lines](https://github.com/decentraland/unity-renderer/blob/ffe59599d71484dfde13a1a85ec365388252f625/unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Billboard/BillboardsController.cs#L56-L58) with just a `IBillboard.LookAt(Vector3)`
- [Here](https://github.com/decentraland/unity-renderer/blob/ffe59599d71484dfde13a1a85ec365388252f625/unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Billboard/BillboardsController.cs#L76-L81) `Tr`, `EntityTransform` and `LastPosition` can be declared as properties in the interface